community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java

```diff
@@ -172,12 +172,6 @@ public final NumberType numberType()
         return NO_NUMBER;
     }
 
-    @Override
-    public final String toString()
-    {
-        return getClass().getSimpleName() + "<" + prettyPrint() + ">";
```

Comment:
If the implementation is always `prettyPrint()`, why not just change it here in the superclass?

Reply:
That is a good point